docs/site/translation/core-data/exemplars.md

```diff
@@ -77,10 +77,16 @@ There are different categories:
-| _punctuation_ | ‐ – — , ; : ! ? . … ‘ ' ’ ′ ″ “ " ” ( ) [ ] / @ &amp; # § † ‡ * | **The punctuation characters** customarily used with your language.<br /><br /> For example, compared to the English list, Arabic might remove ; , ? /, and add ؟ \ ، ؛.<br /><br /> _Don't include purely math symbols such as +, =,   ±, and so on._ |
-| _auxiliary_ | á à ă â å ä ã ā æ ç é è ĕ ê ë ē í ì ĭ î ï ī ñ ó ò ŏ ô ö ø ō œ ú ù ŭ û ü ū ÿ | **Additional letters and punctuation** (beyond the minimal set) used in foreign or technical words found in typical magazines, newspapers, &amp;c.<br /><br /> For example, you could see the name Schröder in English in a magazine, so  ö  is in the set. However, it is very uncommon to see  ł , so that isn't in the auxiliary set for English. Publication style guides, such as  The Economist Style Guide  for English, are useful for this.<br /><br /> If your language has both upper and lowercase letters, only include the lowercase (and İ for Turkish and similar languages). |
+| _main letters_ | a b c d e f g h i j k l m n o p q r s t u v w x y z | **The minimal characters** required for your language (other than punctuation).<br /><br /> The test to see whether or not a letter belongs in the main set is based on whether it is acceptable in your language to always use spellings that avoid that character. For example, English characters do not contain the accented letters that are sometimes seen in words like  résumé  or  naïve , because it is acceptable in common practice to spell those words without the accents.<br /><br />If your language has both upper and lowercase letters, only include the lowercase (and İ for Turkish and similar languages).<br/><br/>_This list may contain combining marks: if they are only used with a small set of letters, include just those sequences; otherwise you can add them as free-standing characters (between spaces)._|
+| _auxiliary_ | á à ă â å ä ã ā æ ç é è ĕ ê ë ē í ì ĭ î ï ī ñ ó ò ŏ ô ö ø ō œ ú ù ŭ û ü ū ÿ | **Additional letters ~~and punctuation~~** (beyond the minimal set) used in foreign, old-fashioned, or technical words found in typical magazines, newspapers, &amp;c.<br /><br /> For example, you could see the name Schröder in English in a magazine, so  ö  is in the set. However, it is very uncommon to see  ł , so that isn't in the auxiliary set for English. Publication style guides, such as  The Economist Style Guide  for English, are useful for this.<br /><br /> If your language has both upper and lowercase letters, only include the lowercase (and İ for Turkish and similar languages). |
```

Comment:
```suggestion
| _auxiliary_ | á à ă â å ä ã ā æ ç é è ĕ ê ë ē í ì ĭ î ï ī ñ ó ò ŏ ô ö ø ō œ ú ù ŭ û ü ū ÿ | **Additional letters and punctuation** (beyond the minimal set) used in foreign, old-fashioned, or technical words found in typical magazines, newspapers, etc.<br /><br /> For example, you could see the name Schröder in English in a magazine, so `ö` is in the set. However, it is very uncommon to see `ł`, so that is not in the auxiliary set for English. Publication style guides, such as "The Economist Style Guide" for English, are useful resources.<br /><br /> If your language has both upper and lowercase letters, only include the lowercase (and İ for Turkish and similar languages). |
```